components/pagination/spec/Pagination.stories.mdx

```diff
@@ -0,0 +1,107 @@
+import { Meta, Preview, Props, Story } from '@not-govuk/docs-components';
+import { Pagination } from '../src/Pagination';
+import readMe from '../README.md';
+
+<Meta
+  title="pagination"
+  component={ Pagination }
+  parameters={ {
+    chromatic: { viewports: [640, 480] },
+    description: 'Component to allow separating long tables into separate tables',
+    jest: ['Pagination'],
+    notes: readMe
+  } }
+/>
+
+# pagination
+
+Component to allow separating long tables into separate tables
+WRITE MORE DESCRIPTION HERE.
```

Comment:
go on then!..

Reply:
Added now!

Reply:
@RobinKnipe are you happy to resolve this thread?